2.5/README.md

```diff
@@ -23,26 +23,111 @@ modules for their web applications. There is no guarantee for any specific npm o
 version, that is included in the image; those versions can be changed anytime and
 the nodejs itself is included just to make the npm work.
 
-Usage
+Usage in Openshift
 ---------------------
-For this, we will assume that you are using the `rhscl/ruby-25-rhel7 image`, available via `ruby:2.5` imagestream tag in Openshift.
+For this, we will assume that you are using the `ubi8/ruby-25 image`, available via `ruby:2.5` imagestream tag in Openshift.
 Building a simple [ruby-sample-app](https://github.com/sclorg/s2i-ruby-container/tree/master/2.5/test/puma-test-app) application
 in Openshift can be achieved with the following step:
 
     ```
-    oc new-app ruby:2.5~https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.5/test/puma-test-app/
+    oc new-app ruby:2.5~https://github.com/sclorg/rails-ex.git
     ```
 
-The same application can also be built using the standalone [S2I](https://github.com/openshift/source-to-image) application on systems that have it available:
+**Accessing the application:**
+```
+$ oc get pods
+$ oc exec <pod> -- curl 127.0.0.1:8080
+```
 
-    ```
-    $ s2i build https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.5/test/puma-test-app/ rhscl/ruby-25-rhel7 ruby-sample-app
-    ```
+Source-to-Image framework and scripts
+-------------------------------------
+This image supports the [Source-to-Image](https://docs.openshift.com/container-platform/3.11/creating_images/s2i.html)
```

Comment:
Aren't those docs quite old?

Reply:
Maybe, but I didn't find such detailed s2i documentation in the later versions and this link is the same as in nodejs and httpd container.

Reply:
That's probably because s2i doesn't work with podman.

Reply:
Depends what you mean by `s2i` here. s2i, the technology, does work even with podman, s2i, the binary, does work with podman as well (via `--as-dockerfile`), but is not packaged anywhere as far as I know. I do not think the binary will ever work natively with podman, Openshift is our primary customer for the technology.

That said, I agree we can at least link to newer docs. E.g. OCP 4.4 version: https://docs.openshift.com/container-platform/4.4/builds/build-strategies.html#images-create-s2i_build-strategies

Reply:
> with podman as well (via `--as-dockerfile`),

Yes, sorry, last time [I've tried](https://github.com/openshift/source-to-image/issues/966#issuecomment-647446739) it didn't work for me:).  But it was at the time of it still being implemented. I've rebuilt s2i it in my COPR etc.. I'll try to reiterate and let you know.

> I do not think the binary will ever work natively with podman

There's a (not sure about the state): `podman system service` 

> That said, I agree we can at least link to newer docs. E.g. OCP 4.4 version: https://docs.openshift.com/container-platform/4.4/builds/build-strategies.html#images-create-s2i_build-strategies

Yes, thanks!